### DISCUSSION TITLE
tsparticles not showing up in header section

### DISCUSSION THREAD

**Discussion by chromaticmoe** [matteobruni/tsparticles#5094] [Q&A]
tsparticles is working fine in 2nd and succeeding sections which has 100vh. but on the 1st section (header), it doesn't. somehow header section fills up the whole 100vh and sits in front of the tsparticles, which shouldn't be because header section should show the tsparticles as background.

I wanted the tsparticles to serve as a background for the whole webpage. Can someone help me pls.

This is my header.tsx:

```
import Button from "@/app/components/button";

const Header: React.FC = () => {
    return ( 
        <div 
            className='
                flex 
                flex-col 
                items-center 
                justify-center 
                h-screen 
                space-y-2 
                sm:space-y-4 
                mx-auto 
                relative
                overflow-hidden'>
            <div className='text-4xl sm:text-6xl font-bold relative text-white'>
                Hi, I'm <span className='text-color-primary relative'>David</span>.
            </div>
            <div className='text-white text-lg sm:text-3xl relative'>Welcome to my page</div>
            <Button />
        </div>
     );
}
 
export default Header;
```

This is my ParticlesBackground.tsx:

```
import { useCallback } from "react";
import type { Container, Engine } from "tsparticles-engine";
import Particles from "react-particles";
import { loadFull } from "tsparticles"; // if you are going to use `loadFull`, install the "tsparticles" package too.
// import { loadSlim } from "tsparticles-slim"; // if you are going to use `loadSlim`, install the "tsparticles-slim" package too.
import particlesConfig from "./particles-config";

const ParticlesBackground = () => {
    const particlesInit = useCallback(async (engine: Engine) => {
        console.log(engine);

        // you can initialize the tsParticles instance (engine) here, adding custom shapes or presets
        // this loads the tsparticles package bundle, it's the easiest method for getting everything ready
        // starting from v2 you can add only the features you need reducing the bundle size
        //await loadFull(engine);
        await loadFull(engine);
    }, []);

    const particlesLoaded = useCallback(async (container: Container | undefined) => {
        await console.log(container);
    }, []);
    return (
        <Particles
            id="tsparticles"
            init={particlesInit}
            loaded={particlesLoaded}
            options={particlesConfig}
        />
    );
};

export default ParticlesBackground;
```

**Comment by matteobruni** [comment]
https://www.reddit.com/r/tsParticles/comments/w8luvb/why_my_particles_are_full_screen_i_want_them_to/

**Reply by chromaticmoe** [reply]
I'm sorry. I've tried everything in the link you've provided but none of it worked. My issue is not related to making tsparticles work in a div. My issue is setting the tsparticles to appear in the header section of my page.

![image](https://github.com/matteobruni/tsparticles/assets/122515859/c62f82de-a8c2-421c-81bd-12d3231cf7e2)

In the image above, I've set each section to take up 100vh of the height and I've scrolled down a bit to show you the next section, which is about. Weirdly enough, the header (one with the Hi, David) doesn't display the tsparticles in the background but the next section, which is About does. And the next section after that which is "work".

![image](https://github.com/matteobruni/tsparticles/assets/122515859/6e341c2d-4553-45e1-bf60-c78ca023184b)
